.gitignore

```diff
@@ -23,6 +23,7 @@ testdir/
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 .idea
+CLAUDE.md
```

Comment:
What is this about?

Reply:
My bad, this shouldn't have been there. Will revert.